packages/web-twig/src/Resources/components/TextField/stories/TextFieldFluid.twig

```diff
@@ -2,7 +2,6 @@
     id="textfieldFluid"
     isFluid
     label="Label"
-    message="Message"
```

Comment:
Replace with `validationText`?

Reply:
This story is supposed to show the Fluid version. So I tried to showcase only that. If I would add validationText, I would have to add a validationState too. And I don't think it is necessary a if so, I would prefer to have it as a second example in Fluid Section.